articles/api-auth/dynamic-client-registration.md

```diff
@@ -0,0 +1,236 @@
+
+- To authenticate users using [Lock](/libraries/lock), you will have to use a version greater than `10.7`.
+
+- They cannot skip user consent when consuming APIs. This is for security purposes, as anyone can create a client, but each client relays on the final user to provide consent.
+
+- They cannot use [ID tokens](/tokens/id-token) to invoke [Management APIv2](/api/management/v2) endpoints. Instead, they should get a Management APIv2 Token (see the *How to get a Management APIv2 Token* panel for details). Note that the client should be granted the `current_user_*` scopes, as required by each endpoint.
+  - `read:current_user`: [List or search users](/api/management/v2#!/Users/get_users), [Get a user](/api/management/v2#!/Users/get_users_by_id), [Get user Guardian enrollments](/api/management/v2#!/Users/get_enrollments)
+  - `update:current_user_metadata`: [Update a user](/api/management/v2#!/Users/patch_users_by_id), [Delete a user's multifactor provider](/api/management/v2#!/Users/delete_multifactor_by_provider)
+  - `delete:current_user_metadata`: [Delete a user](/api/management/v2#!/Users/delete_users_by_id)
+  - `create:current_user_metadata`: [Create a user](/api/management/v2#!/Users/post_users)
+  - `create:current_user_device_credentials`: [Create a device public key](/api/management/v2#!/Device_Credentials/post_device_credentials)
+  - `delete:current_user_device_credentials`: [Delete a device credential](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id)
+  - `update:current_user_identities`: [Link a user account](/api/management/v2#!/Users/post_identities), [Unlink a user identity](/api/management/v2#!/Users/delete_provider_by_user_id)
+
+::: panel-info How to get a Management APIv2 Token
+In order to access the [Management APIv2](/api/management/v2) endpoints from a third party client, you need a Management APIv2 Token. To get one you can use any of the [API Authorization Flows](/api-auth), with the following request parameters:
+- `audience=https://${account.namespace}/api/v2/`
+- `scope=read:current_user update:current_user_metadata`
+:::
+
+## Enable dynamic registration
+
+In this section we will see how you can enable the dynamic registration feature for your tenant.
+
+By default, the feature is disabled for all tenants. To change this, you have to update some account settings, promote the connections you will use with your dynamic clients to **domain connections**, and update your client's login page (if you use Lock).
+
+::: panel-warning Security warning
+Auth0 supports Open Dynamic Registration, which means that if you enable this feature, **anyone** will be able to create clients in your tenant without a token.
+:::
+
+### Update tenant settings
+
+In order to enable the feature, you need to set to `true` the following flags at your tenant's settings:
+- `enable_dynamic_client_registration`: Enables the feature.
+- `enable_pipeline2`: Enables the underlying pipeline that is a required dependency.
```

Comment:
Note to future self: This flag (`enable_pipeline2`) will be removed in the future. Let's keep on the radar that this doc will need to be updated once we finally merge the pipeline.